### DISCUSSION TITLE
Usage of ImageMagick for monochrome bmp image resize

### DISCUSSION THREAD

**Discussion by rmshub** [ImageMagick/ImageMagick#6379] [General]
**Platform**: Embedded linux ARM
I would like to create a thumbnail while selection and image preview while processing on 7inch LCD display. The build system supports only CMake. Could you please share the procedure or files to be complied for magick convert feature.

**Comment by rmshub** [comment]
Could you please help me about this topic

**Comment by snibgo** [comment]
As far as I know, ImageMagick can't currently be built by the CMake build environment.

But someone may know different.

For building from source, see:
https://imagemagick.org/script/install-source.php

For building IM with CMake, see:
https://github.com/ImageMagick/ImageMagick/issues/743
https://github.com/conan-io/conan-center-index/pull/10003
